app/code/Magento/Cms/Block/Widget/Block.php

```diff
@@ -124,16 +119,13 @@ private function getBlock(): ?CmsBlock
         $blockId = $this->getData('block_id');
 
         if ($blockId) {
-            try {
-                $storeId = $this->_storeManager->getStore()->getId();
-                /** @var \Magento\Cms\Model\Block $block */
-                $block = $this->_blockFactory->create();
-                $block->setStoreId($storeId)->load($blockId);
-                $this->block = $block;
-
-                return $block;
-            } catch (NoSuchEntityException $e) {
-            }
+            $storeId = $this->_storeManager->getStore()->getId();
```

Comment:
@fredden I think this change could have influence on current implementation. Or you are sure that NoSuchEntityException never is thrown?

Reply:
I have restored the existing code here to ensure that this pull request does not inadvertently change behaviour. I initially removed it because the linter detected the empty `catch` block and complained. I didn't see a good reason at the time to keep the block. I agree that removing it is a behaviour change which is out of scope for this particular pull request. I'll let others fix the code when they are ready to do so. Thanks for highlighting this out-of-scope change.